### ISSUE TITLE
/f desc (Color issue)

### ISSUE THREAD

**Issue by CheatSupport** [drtshock/Factions#797]
So at: FactionsUUID/blob/1.6.x/src/main/java/com/massivecraft/factions/cmd/CmdDescription.java

We see on line 37 to 39, there is code preventing /f desc &(color code goes here)
But there is for some reason other ways around it (very odd?)

If you do: /f desc <red>Text here
It prints out the description as red (known as &4)

Also if you do: /f desc '4
It prints out the description as red (also known as &4)

In other words, you can /f desc <color> and /f desc 'colorcode
To get colors in f desc. If you read line 50 in CmdDescription.java, it shows: "// players can inject "&" or "`" or "<i>" or whatever in their description;"

Not sure why that is, but please fix it. Thanks!

**Comment by CheatSupport** [comment]
Please fix :) 👍

**Comment by Kakifrucht** [comment]
That doesn't seem to be ocurring to me.
